pkg/middleware/auth_proxy/auth_proxy.go

```diff
@@ -179,12 +180,12 @@ func (auth *AuthProxy) Login(logger log.Logger, ignoreCache bool) (int64, *Error
 	}
 
 	if isLDAPEnabled() {
-		id, err := auth.LoginViaLDAP()
-		if err != nil {
-			if err == ldap.ErrInvalidCredentials {
+		id, e := auth.LoginViaLDAP()
```

Comment:
err is renamed to e since it's not of the error type, which is expected of variables named err.